CONFIGDOC.md

```diff
@@ -8,15 +8,15 @@ You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-con
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
 ## Use MATLAB for Build
-Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox appears in the **Build Environment** section of the project configuration window.
+Once you install the plugin on Jenkins, **Use MATLAB version** appears in the **Build Environment** section of the project configuration window.
 
 ![matlab_path](https://user-images.githubusercontent.com/48831250/76883191-1c528b00-6852-11ea-9242-7d143f8a5c46.png)
 
-Select the **Use MATLAB version** checkbox to specify the full path to the MATLAB root folder that should be used for the build. The information in the **MATLAB root** box enables the plugin to prepend MATLAB to the PATH environment variable of the build agent. 
+Select **Use MATLAB version** to specify the full path to the MATLAB root folder that should be used for the build. The information in the **MATLAB root** box enables the plugin to prepend MATLAB to the PATH environment variable of the build agent. 

```

Comment:
Perhaps "to prepend MATLAB to the **system** PATH environment variable...". 

Since MATLAB has its own path, I find it useful to be as explicit as possible when talking about the system path.

Reply:
Done.